Chat-Commands.md

```diff
@@ -75,76 +90,98 @@ Format: `!kc [boss]`, `!kc [activity]`
-`cerb` -> `Cerberus`  
-`supreme` -> `Dagannoth Supreme`  
-`rex` -> `Dagannoth Rex`  
-`prime` -> `Dagannoth Prime`  
-`wt` -> `Wintertodt`  
-`barrows` -> `Barrows Chests`  
-`cg` -> `Corrupted Gauntlet`  
-`dusk`, `dawn`, `gargs` -> `Grotesque Guardians`  
-`smoke devil`, `thermy` -> `Thermonuclear Smoke Devil`  
-`zuk`, `inferno`, -> `TzKal-Zuk`  
+`phantom`, `muspah`, `pm` -> `Phantom Muspah`  
+`nm`, `tnm`, `nmare`, `the nightmare` -> `Nightmare`  
+`pnm`, `phosani`, `phosanis`, `phosani nm`, `phosani nightmare`, `phosanis nightmare` -> `Phosani's Nightmare`  
 `sara`, `saradomin`, `zilyana`, `zily` -> `Commander Zilyana`  
 `zammy`, `zamorak`, `kril`, `kril trutsaroth` -> `K'ril Tsutsaroth`  
 `arma`, `kree`, `kreearra`, `armadyl` -> `Kree'arra`  
 `bando`, `bandos`, `graardor` -> `General Graardor`  
-`nm`, `tnm`, `nmare`, `the nightmare` -> `Nightmare`  
-`cox`, `xeric`, `chambers`, `olm`, `raids` -> `Chambers of Xeric`  
-`cox cm`, `xeric cm`, `chambers cm`, `olm cm`, `raids cm` -> `Chambers of Xeric: Challenge Mode`  
-`tob`, `theatre`, `verzik`, `verzik vitur`, `raids 2` -> `Theatre of Blood`    
-`tob cm`, `tob hm`, `htm` -> `Theatre of Blood: Hard Mode`  
-`toa` -> `Tombs of Amascut`  
-`toa expert`, `toa expert mode`, `tombs of amascut - expert` -> `Tombs of Amascut: Expert Mode`  
+`supreme` -> `Dagannoth Supreme`  
+`rex` -> `Dagannoth Rex`  
+`prime` -> `Dagannoth Prime`  
 `duke` -> `Duke Sucellus`  
-`duke awakened`, `duke sucellus awakened` -> `Duke Sucellus (awakened)` 
+`duke awakened`, `duke sucellus awakened` -> `Duke Sucellus (awakened)`  
 `levi`, `the leviathan` -> `Leviathan`  
-`levi awakened`, `leviathan awakened`, `the leviathan awakened` -> `Leviathan (awakened)`
+`levi awakened`, `leviathan awakened`, `the leviathan awakened` -> `Leviathan (awakened)`  
 `vard` -> `Vardorvis`  
 `vard awakened`, `vardorvis awakened` -> `Vardorvis (awakened)`  
 `wisp`, `whisp`, `the whisperer` -> `Whisperer`  
-`wisp awakened`, `whisp awakened`, `whisperer awakened` -> `Whisperer (awakened)`
+`wisp awakened`, `whisp awakened`, `whisperer awakened` -> `Whisperer (awakened)`  
+`barrows` -> `Barrows Chests`  
+`gaunt`, `the gauntlet` -> `Gauntlet`  
+`cg`, `cgaunt`, `cgauntlet`, `the corrupted gauntlet` -> `Corrupted Gauntlet`  
+`jad`, `tzhaar fight cave` -> `TzTok-Jad`  
+`zuk`, `inferno` -> `TzKal-Zuk`  
+`jad 1-6` -> `TzHaar-Ket-Rak's [First-Sixth] Challenge`  
+`cox`, `xeric`, `chambers`, `olm`, `raids` -> `Chambers of Xeric`  
+`cox solo`, `cox duo`, `cox [1-24]`, `cox 24+` -> `Chambers of Xeric [1-24+] players`  
+`cox cm`, `xeric cm`, `chambers cm`, `olm cm`, `raids cm`, `chambers of xeric - challenge mode` -> `Chambers of Xeric: Challenge Mode`  
+`cox cm solo`, `cox cm duo`, `cox cm [1-24]`, `cox cm 24+` -> `Chambers of Xeric: Challenge Mode [1-24+] players`  
+`theatre of blood: story mode`, `tob sm`, `tob story mode`, `tob story`, `tob entry mode`, `tob em`, `tob entry` -> `Theatre of Blood: Entry Mode`  
```

Comment:
```suggestion
`tob sm`, `tob story mode`, `tob story`, `tob entry mode`, `tob em`, `tob entry` -> `Theatre of Blood: Entry Mode`  
```
my only nit is when documenting all variations like this, we end up with a giant list that wraps, which feels unnecessary when there's already a reasonable list of examples

this happens similarly with seers, but there's *_so_* many for that one that it wrapped 2 lines and doesn't look as bad

Reply:
For me it only wraps for Seers (which could arguably be shortened, since it includes a lot of the same stuff + ``agility``) and ToB: EM. I personally don't mind it, and this is the only way for regular users to find out the aliases. Is it bad enough to (somewhat randomly) remove an alias?

Reply:
Mainly I don't see the point of an alias that's 28 characters long, while it was accepted in the code it doesn't feel that bad to "forget" to tell users about this extremely long alias